src/ib_module.F

```diff
@@ -1448,7 +1448,7 @@ subroutine ib_side_weno(stag,ix,jy,kz,c1,c2,rru,rrv,dumx,dumy,a,hadvorder,bndy,h
   ELSEIF( stag.eq.4 )THEN
 
   if( doit )then
-    !$acc parallel loop private(i,j,k,ubar,vbar,cc1,cc2)
+    !$acc parallel loop default(present) private(i,j,k,ubar,vbar,cc1,cc2)
```

Comment:
maybe add `gang vector` clause and `collapse` clause, and remove `private` clause?